tests/suites/cassandra_externalservice/external_service_test.go

```diff
@@ -0,0 +1,132 @@
+package cassandra_externalservice
```

Comment:
Does it make sense to put this in `tests/suites/external_service/external_service_test.go` to maintain consistency? the "cassandra_" prefix feels extraneous...

Reply:
```suggestion
package external_service
```